homeassistant/components/unifi/__init__.py

```diff
@@ -59,6 +60,20 @@ async def async_unload_entry(hass, config_entry):
     return await controller.async_reset()
 
 
+async def async_migrate_entry(hass, config_entry):
+    """Migrate old entry."""
+    LOGGER.debug("Migrating from version %s", config_entry.version)
+
+    #  Flatten configuration but keep old data if user rollbacks HASS prior to 2021.03
```

Comment:
This won't work, the old version will not accept version 2, and will fail to setup.
If the old version accepts the extra data, add that without bumping the version instead, then after a few versions bump versions and discard the extra data.

Reply:
Thanks! Will do as suggested

Reply:
Done